### PR TITLE
Update UGDG for Wk11

### PR DESCRIPTION
**Pull request by JLoh579**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -593,7 +607,7 @@ shopping lists and also providing helpful features like budget tracking
 |v2.0|practical home cook|be able to edit the items in my lists|update the items on my list accordingly|
 |v2.0|organised home cook|save my list|have a local copy of my list|
 |v2.0|organised home cook|load my saved list|add on to my existing list|
-|v2.0|frugal home cook|see the remaining budget update based on the quantity of items|see how much I spend based on how much I buy|
+|v2.0|frugal home cook|see the remaining budget update based on the quantity of items|see how much I spend based on ````````how much I buy|
```
Comment (chaosInTheWild) [inline docs/DeveloperGuide.md]: There seems to be a formatting issue here, at the last column.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -295,31 +302,33 @@ having to edit the mark and unmark feature difficult.
...
 <code>Budget</code> object to find the current budget and the remaining budget. The results are then printed to console.
 
 The following sequence diagrams below show how the display feature works. Note the <code>Ui</code> class is
 omitted to emphasise the other classes:
 
-![alt text](images/ListFeature.png)
+![alt text](images/List.png)
 
-![alt text](images/ListFeature_SD.png)
+![alt text](images/List_SD.png)
```
Comment (chaosInTheWild) [inline docs/DeveloperGuide.md:327]: These two diagrams are not appearing in the developer guide for some reason. Might have to fix that.

**Review by chaosInTheWild — COMMENTED** [review]
Generally, it looks okay, just that need to fix some formatting errors and missing diagrams. 

Also, the CI tests are failing for the UNIX redirection test. Might want to look into that too!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (JLoh579) [reply docs/DeveloperGuide.md]: Oh yea, thanks!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (JLoh579) [reply docs/DeveloperGuide.md:327]: Oh yea, forgot that I changed the names of the diagrams, thanks!

**Pull request closed without merging**